.github/labeler.yml

```diff
@@ -0,0 +1,112 @@
+aws:apigateway:
+  - localstack/aws/api/apigateway/*.py
+  - localstack/services/apigateway/*.py
+  
+aws:lambda:
+  - localstack/aws/api/awslambda/*.py
+  - - localstack/services/awslambda/*.py
+  
+aws:cloudformation:
+  - localstack/aws/api/cloudformation/*.py
+  - localstack/utils/cloudformation/*.py
+  - - localstack/services/cloudformation/*.py
+  
+aws:cloudwatch:
+  - localstack/aws/api/cloudwatch/*.py
+  - localstack/utils/cloudwatch/*.py
+  - localstack/services/cloudwatch/*.py
+  
+config:
+  - localstack/aws/api/config
+  - localstack/services/configservice/*.py
+  
+aws:dynamodb:
+  - localstack/aws/api/dynamodb/*.py
+  - localstack/aws/api/dynamodbstreams/*.py
+  - localstack/services/apigateway/*.py
+  
+aws:ec2:
+  - localstack/aws/api/ec2/*.py
+  - localstack/services/ec2/*.py
+  
+aws:es:
+  - localstack/aws/api/es/*.py
+  - localstack/services/es/*.py
+  
+aws:firehose:
+  - localstack/aws/api/firehose/*.py
+  - localstack/services/firehose/*.py
+  
+aws:iam:
+  - localstack/aws/api/iam/*.py
+  - localstack/services/iam/*.py
+  
+aws:kms:
+  - localstack/aws/api/kms/*.py
+  - localstack/services/kms/*.py
+  
+aws:logs:
+  - localstack/aws/api/logs/*.py
+  - localstack/services/logs/*.py
+  
+aws:opensearch:
+  - localstack/aws/api/opensearch/*.py
+  - localstack/services/opensearch/*.py
+  
+aws:redshift:
+  - localstack/aws/api/redshift/*.py
+  - localstack/services/redshift/*.py
+
+aws:route53:
+  - localstack/aws/api/route53/*.py
+  - localstack/services/route53/*.py
+  
+aws:s3:
+  - localstack/aws/api/s3control/*.py
+  - localstack/services/s3/*.py
+  - localstack/services/s3control/*.py
+  
+aws:ses:
+  - localstack/aws/api/ses/*.py
+  - localstack/services/ses/*.py
+  
+aws:sns:
+  - localstack/aws/api/sns/*.py
+  - localstack/services/sns/*.py
+  
+aws:sqs:
+  - localstack/aws/api/sqs/*.py
+  - localstack/services/sqs/*.py
+  
+aws:ssm:
+  - localstack/aws/api/ssm/*.py
+  - localstack/services/ssm/*.py
+  
+aws:stepfunctions:
+  - localstack/aws/api/stepfunctions/*.py
+  - localstack/services/stepfunctions/*.py
+  
+aws:sts:
+  - localstack/aws/api/sts/*.py
+  - localstack/services/sts/*.py
+  
+aws:kinesis:
```

Comment:
nit: Could be moved further up, to preserve the alphabetical order of service names.